packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -0,0 +1,51 @@
+import { consoleLogger } from '@monorepo/common';
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import type { UtilsConfig } from '../config';
+import { setUtilsConfig } from '../config';
+import type { InitAPIDiscovery } from '../types/api-discovery';
+import { initAPIDiscovery } from './api-discovery';
+import { useReduxStore } from './redux';
+
+type AppInitSDKProps = {
+  /** Only child is your Application */
+  children: React.ReactElement;
+  configurations: {
+    apiDiscovery?: InitAPIDiscovery;
+    appFetch: UtilsConfig['appFetch'];
```

Comment:
Hard disagree. We need to keep this API. Let us not take everything we do internal and push it outwards... this is public API, we can store a ton of things in config if we want and it should not be exposed just because it's in our config.

Reply:
So we can make two types now, and do like a `utilsConfig: PublicUtilsConfig` I guess?

Reply:
@andrewballantyne My bad, I thought `UtilsConfig` is the public API but the actual public API is `AppInitSDK` component and its props.